src/formatDate.js

```diff
@@ -51,6 +52,63 @@
 
 function formatDate(date, fromFormat, toFormat) {
   // write code here
+  const res = [];
+  const currentYear = new Date().getFullYear() % 100;
+  const dateArr = date.split(`${fromFormat[fromFormat.length - 1]}`);
+  const dateObg = {
+    year: null,
+    month: null,
+    day: null,
+    separator: null,
+  };
```

Comment:
```suggestion
  const dateObg = {};
```
looks like redundant to make each key null. we can start with empty object